test-app/app_sim.c

```diff
@@ -58,6 +58,7 @@ int do_cmd(const char *cmd)
         return 1;
     }
     if (strcmp(cmd, "get_version") == 0) {
+        sleep(.5);
```

Comment:
The sleep is to make debugging or watching output easier?

Reply:
definitely a debug leftover. I will fix.